### PR TITLE
Autofac decoration

### PR DESCRIPTION
**Pull request by AntsiferauGodel**
using autofac to decorate classes with loggers

### CODE REVIEW

File: Unit-03/lab-file-storage/ConsoleFileStorage/ConsoleFileStorage.Business/Factories/CommandFactory.cs [hunk 1]
```diff
@@ -30,7 +30,7 @@ public CommandFactory(IManager manager)
                 new UserInfoCommand(),
                 new CommandListCommand(),
                 new FileExportCommand(),
-                new SetupCurrentUserCommand()
+                new SetupCurrentUserCommand(),
```
Comment (HannaKeryna) [inline Unit-03/lab-file-storage/ConsoleFileStorage/ConsoleFileStorage.Business/Factories/CommandFactory.cs]: Please remove extra comma

File: Unit-03/lab-file-storage/ConsoleFileStorage/ConsoleFileStorage.Business/Managers/Manager.cs [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
 using FileNotFoundException = ConsoleFileStorage.Data.Exceptions.FileNotFoundException;
 
-namespace ConsoleFileStorage.Business
+namespace ConsoleFileStorage.Business.Managers
 {
     public class Manager : IManager
     {
-        private readonly string workingDirectory;
+        private readonly StringBuilder workingDirectory = new StringBuilder();
         private readonly IMetaManager metaManager;
```
Comment (HannaKeryna) [inline Unit-03/lab-file-storage/ConsoleFileStorage/ConsoleFileStorage.Business/Managers/Manager.cs]: Let's discuss about StringBuilder

File: Unit-03/lab-file-storage/ConsoleFileStorage/ConsoleFileStorage.Business/Factories/CommandFactory.cs (hunk 1 above)
Comment (AntsiferauGodel) [reply Unit-03/lab-file-storage/ConsoleFileStorage/ConsoleFileStorage.Business/Factories/CommandFactory.cs]: removed

File: Unit-03/lab-file-storage/ConsoleFileStorage/ConsoleFileStorage.Business/Managers/Manager.cs (hunk 2 above)
Comment (AntsiferauGodel) [reply Unit-03/lab-file-storage/ConsoleFileStorage/ConsoleFileStorage.Business/Managers/Manager.cs]: moved to string

**Pull request merged**